tests/test_formatutils.py

```diff
@@ -48,6 +52,40 @@ def test_get_fstr_args():
     return results
 
 
+@pytest.mark.parametrize(('sample', 'expected'), zip(_TEST_TMPLS, [
+        ([], [('hello', str)]),
+        ([], [('hello', str)]),
+        ([], [('hello', str), ('width', str)]),
+        ([], [('hello', str), ('fchar', str), ('width', str)]),
+        ([(0, str), (1, int), (2, float)], []),
+        # example 6 is skipped
+]))
+def test_get_format_args(sample, expected):
+    """Test `get_format_args` result as tuples."""
+    assert get_format_args(sample) == expected
+
+
+@pytest.mark.parametrize(('sample', 'expected'), zip(_TEST_TMPLS, [
```

Comment:
I assume so; the pytest docs show an [example](http://docs.pytest.org/en/latest/parametrize.html) of a multi-line parametrize right at the top. There are also a few instances in the pytest tests themselves (such as in *test_conftest.py*, *test_config.py* and *test_mark.py*).

If I have something like this that I intend to reuse exactly as-is, rather than moving the list of samples to a separate variable, I create a new mark:
```
pytest.mark.get_format_samples = pytest.mark.parametrize(('sample', 'expected), ...
```